playbooks/roles/ocp-cluster-etcd-operator/README.md

```diff
@@ -0,0 +1,53 @@
+ocp-cluster-etcd operator
+=========================
+
+The control plane hardware speed values are "Standard", "Slower", or the default, which is "".
+The default setting allows the system to decide which speed to use. This value enables upgrades from versions where this feature does not exist, as the system can select values from previous versions
+
+By selecting one of the other values, you are overriding the default. If you see many leader elections due to timeouts or missed heartbeats and your system is set to "" or "Standard", set the hardware speed to "Slower" to make the system more tolerant to the increased latency
+
+This playbook will handle below:
+- It will check the DaemonSet for HEARTBEAT_INTERVAL and LEADER_ELECTION_TIMEOUT parameters
+- Change the control plane hardware speed tolerance to "Slower"
+- Run the acme-air workload and e2e conformance test suite
+- Also it will run the kube-burner workload test
```

Comment:
Can we create a separate role for kube-burner workload test ? if required we can reuse this role.

Reply:
Can we make it optional to deploy acme-air app and run e2e/kube-burner workload tests ? Or If we are going to run these roles every time, then we can keep it.

Reply:
Added separate role for kube-burner workload and made deploy acme-air app and run e2e/kube-burner workload as optional